modules/olm-mirroring-catalog.adoc

```diff
@@ -10,8 +10,8 @@ You can mirror the Operator contents of a Red Hat-provided catalog, or a custom
 
 [IMPORTANT]
 ====
-* The internal registry of the {product-title} cluster cannot be used as the target registry because it does not support pushing without a tag, which is required during the mirroring process.
-* Running `oc adm catalog mirror` might result in the following error: `error: unable to retrieve source image`. This error occurs when image indexes include references to images that no longer exist on the image registry. Image indexes might retain older references to allow users running those images an upgrade path to newer points on the upgrade graph. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed]. 
+* The {product-registry} of the {product-title} cluster cannot be used as the target registry because it does not support pushing without a tag, which is required during the mirroring process.
```

Comment:
Same comment as before wrt using both `{product-registry}` and `{product-title}`.